src/main/java/org/spongepowered/api/stats/StatisticGroups.java

```diff
@@ -30,7 +30,14 @@
  */
 public class StatisticGroups {
 
+    /**
+     * Group for statistics that do not match in any group.
+     */
     public static final StatisticGroup GENERAL = null;
+    /**
+     * Statistic counting the distances moved in a specific way.
+     */
+    public static final StatisticGroup DISTANCE_MOVED = null;
```

Comment:
I have seen the picture before (the last time you? quoted it)
I know that you are referencing the general view in minecraft, but this entire group has a mayor subgroup that is worth putting in a own subgroup. (Because it may be split in a extra view in minecraft itself)

I considered an improvement. But its a helper group so it is optional.